### PR TITLE
 Interop

### PR DESCRIPTION
**Pull request by tanneberger**

### CODE REVIEW

File: tests/test_live_fetching.c [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+	 */
+
+	//char RPKI_CACHE_HOST[] = "rpki-validator.realmv6.org";
+	//char RPKI_CACHE_PORT[] = "8283";
+
+	// REPLACE THIS BY YOUR RTR SERVER
+	char RPKI_CACHE_HOST[] = "rtrlab.tanneberger.me";
+	char RPKI_CACHE_PORT[] = "3325"; // rir rtr
+	//char RPKI_CACHE_PORT[] = "3324"; // aspa rtr
+
```
Comment (c-h-a-r-l-y) [inline tests/test_live_fetching.c:61]: not sure if we need that comment

**Review by carl-gh — APPROVED** [review]
looks good, ready to merge

File: tests/test_live_fetching.c (hunk 1 above)
Comment (carl-tud) [inline tests/test_live_fetching.c:61]: do we need that comment?

**Review by carl-tud — APPROVED** [review]

File: tests/test_live_fetching.c (hunk 1 above)
Comment (moversity) [reply tests/test_live_fetching.c:61]: i believe we don't

File: tests/test_live_fetching.c [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+	/* printing all fetched objects */
+	struct aspa_array* array = ((*groups[0].sockets)->aspa_table->store)->aspa_array;
+
+	printf("ASPA:\n");
+	for (uint32_t i = 0; i < array->size; i++) {
+		printf("CAS %i -> ", array->data[i].customer_asn);
+		for (uint32_t j = 0; j < array->data[i].provider_count; j++) {
+			printf("%i ", array->data[i].provider_asns[j]);
+		}
+		printf("\n");
```
Comment (moversity) [inline tests/test_live_fetching.c]: let's use the format from rtrclient and krillc here:
`ASPA %u => [ %u, %u... ]`

Comment (moversity) [reply tests/test_live_fetching.c]: i implemented this in 9b21dc5

**Pull request merged**